docs/README.md

```diff
@@ -1,12 +1,25 @@
-# Scylla documentation
+# ScyllaDB Documentation
```

Comment:
@tzach @dorlaor do we have an official policy on what do we call "Scylla" and what "ScyllaDB"?
In the past I was under the impression that we are calling our database **Scylla** and the company **ScyllaDB**, but I'm not sure this is still our policy.
By the way, after we decide on a policy, let's update https://en.wikipedia.org/wiki/Scylla_(database) to conform to it, and not use "Scylla" in one paragraph and "ScyllaDB" in the next paragraph...

Reply:
Turns out I may be out of date....

In https://en.wikipedia.org/wiki/Talk:Scylla_(database) Peter Corless wrote:
"Hello folks! Peter Corless from ScyllaDB here. We've heard plenty of feedback and confusion over the years as to whether it should just be "Scylla" or "ScyllaDB" and we're bowing to popular demands to change it all to "ScyllaDB.""

Is this the official decision now?

Reply:
@nyh Not long ago, I was told that "ScyllaDB" now refers to both the company and the product. We're using "ScyllaDB" referring to the database when adding new or updating existing content. The same goes for marketing materials and other kinds of content. For example, see https://www.scylladb.com/product/.

Reply:
@nyh yes, its official, and we try to use ScyllaDB where we can, at least moving forward
There are no plans to change the repo name (for example) for now

Reply:
Interesting and surprising - because as I said I was under the impression that the decision was opposite :-) Take a look at our top-level README.md - https://github.com/scylladb/scylla/ - the project is called "Scylla" everywhere, not "ScyllaDB". Its github project name is also "scylla". Wikipedia still has https://en.wikipedia.org/wiki/Scylla_(database). 
I'll start by renaming the Wikipedia article now.

Reply:
I fixed the Wikipedia entry. It is now "ScyllaDB".

Reply:
I suppose we should rename the repository to scylladb.git (github will maintain a redirect from scylla.git), rename the binary to /usr/bin/scylladb (with a compatibilty symlink /usr/bin/scylla) the rpb/deb packages (with Obsoletes: scylla-server), and the systemd .service files (with dummy scylla-server.service that just call scylladb-server.service).

Reply:
Just make sure to update the docs with each change.
Lucky for us, its easier to do now.